content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -628,3 +628,40 @@ If the :abbr:`PA (Public Administration)` business doesn't reply within 15 days,
 the :abbr:`PA (Public Administration)` business directly, sending them both the invoice and the
 received deadline notification by email. You can make an arrangement with them and manually set the
 correct :guilabel:`SdI State` on your invoice.
+
+Point of Sale in Italy: Fiscal printer
+======================================
+
+Starting with Odoo 18.0, we introduced the support for Italian fiscal printers for your point of
+sale.
+These printers often come with the default setting to use HTTP. In order for such a printer to work
+with Odoo, that setting has to be changed to enable the printer to support HTTPS.
+
+EPSON FP90-III
+--------------
+
+To setup this printer, you will need the configuration software EpsonFPWizard or the keyboard
+connected to the printer. With the keyboard, you will then need to perform the following:
+
+#. Make sure you are on the default state of your fiscal printer
+#. Type **3333**
+#. Press **Chiave**
+#. Type **34**
+#. Press **Contante** until the screen displays **SSL**
+#. Press X to turn the value **0** into a **1**
+#. Press **Contante** 3 times to confirm
+#. Press **Chiave**
```

Comment:
I made some suggestions in the following list:
- to avoid the unnecessary "you/your" 
- :guilabel: markup should be used to identify any text of the interactive user interface and literal to write technical terms or a specific value to insert.
- we usually try to start with the intent ("To do this, do yyy, and select zzz (for example)).

Also, as we tend to be as precise as possible in the instructions, could you add the name of the fields/windows where the information is to be typed or clicked whenever possible? Thank you!
```suggestion
#. Make sure that the fiscal printer is set to its default state.
#. Type `3333`.
#. Click :guilabel:`Chiave`.
#. Type `34`.
#. To display :guilabel:`SSL` on the screen, click :guilabel:`Contante` repeatedly.
#. To turn the value `0` into a `1`, click :guilabel:`X`.
#. To confirm, click :guilabel:`Contante` 3 times.
#. Click :guilabel:`Chiave`.
```

Reply:
Hi @viso-odoo Thanks for the clarification. You're right, I think that it won't be possible to be more specific then. But, yes, I'll be in GR3 tomorrow morning, I'll contact you so you can show me! Thank you :)